webpack.config.development.js

```diff
@@ -8,7 +8,13 @@ config.devServer = {
     ]
   },
   hot: true,
-  inline: true
+  inline: true,
+  proxy: {
+    '/api': {
+      changeOrigin: true,
+      target: 'http://localhost:3000'
```

Comment:
Actually, I'm going to hold off on that. Principally, because I don't really know how off the top of my head.